### ISSUE TITLE
after update to sm 2.4.1 ... no keywords shown of new products :(

### ISSUE THREAD

**Issue by MariaPho** [graphpaperpress/Sell-Media#826]
Hi
I update to the newer Version of SellMedia. But now, from all new created products, the keywords are no more shown on the product page. From the existing products from before the update, the keywords are shown.
in the media-section of the new products I can see, that they keywords indeed were imported - so they are there ...but not shown.
(in template-tag, the conditions fail here:
if ( wp_get_post_terms( $post_obj->ID, 'keywords' ) && ! get_query_var( 'id' ) ) {   )

So I cant really go Forward from here.

I thought about going back to 2.33 ....but there, the contributors somehow are not saved ( I read it is a bug).  Probably still best, I could fix that bug and stop updating...

because....

it is really annoying to have at least one significant bug in every Version, that stops you from going Forward. I really consider leaving sellmedia.

**Comment by erikksson** [comment]
....I can unterstand your fustration .... I share it with you.
I would not put too much hope on support from the vendor. As you can see, further down the list within the open issues  I reported the exact same keyword problem already in May.
that means: not a single feedback for an obvious bug within 4 months. (at least, you confirm, that it really exists).
A photo selling platform without keywords functioning is not usable. I am testing different other products right now. Will report back here with some conclusions - so maybe you get at least any Input or help here.

**Comment by thadallender** [comment]
In version 2.4, Keywords are now associated with individual MEDIA, not ITEMS in Sell Media. This is because ITEMS can contain galleries of photos. Keywords must be a one-to-one ratio, not one-to-many, otherwise they're useless.

So, check your Media Library and you'll see the Keywords taxonomy there for both existing and new uploads.

**Comment by MariaPho** [comment]
yes, thanks, the keywords are at the individual media, I noticed that.
but sell media doesn't show any keywords anymore on the individual product pages for any new products.  and because the keywords are indeed there within the media, it must be some sellmedia problem

i did not Change anything in the configuration, I just updated .... 

While existing Uploads still Show the keywords on the product page, the new ones don't.
